tools/windows-node-installer/README.md

```diff
@@ -42,7 +42,7 @@ Available Commands:
 ```bash
 ./wni aws create --kubeconfig <path to OpenShift cluster>/kubeconfig --credentials <path to aws>/credentials 
 --credential-account default --image-id ami-06a4e829b8bbad61e --instance-type m4.large --ssh-key <name of the 
-existing ssh key, ie: libra>
+existing ssh key, ie: libra> --private-key <private key to decrypt the aws instance password>
```

Comment:
If libra is being EOL then shouldnt it be changed here?

Reply:
That is good point, @sebsoto. @ravisantoshgudimetla I think we should just remove `libra`. This is internal stuff.

Reply:
I think it should be a separate PR or if you want badly I can do it in a separate commit within this PR.

Reply:
I agree. You should then also remove the change related to ```.